shell/browser/serial/serial_chooser_context.cc

```diff
@@ -43,35 +44,56 @@ std::string EncodeToken(const base::UnguessableToken& token) {
       std::string_view(reinterpret_cast<const char*>(&data[0]), sizeof(data)));
 }
 
+base::UnguessableToken DecodeToken(std::string_view input) {
+  std::string buffer;
+  if (!base::Base64Decode(input, &buffer) ||
+      buffer.length() != sizeof(uint64_t) * 2) {
+    return base::UnguessableToken();
+  }
+
+  const uint64_t* data = reinterpret_cast<const uint64_t*>(buffer.data());
+  std::optional<base::UnguessableToken> token =
+      base::UnguessableToken::Deserialize(data[0], data[1]);
+  if (!token.has_value()) {
+    return base::UnguessableToken();
+  }
+  return token.value();
+}
+
```

Comment:
`DecodeToken()` doesn't seem to be called in this PR?

Reply:
that'd be because it shouldn't be there - removed it